parsec/api/data/base.py

```diff
@@ -70,13 +61,17 @@ def __new__(cls, name, bases, nmspc):
             nmspc["SCHEMA_CLS"], DataValidationError, DataSerializationError
         )
 
-        return cls.CLS_ATTR_COOKING(raw_cls)
```

Comment:
Why the removing ? (was it too much magic for mypy ?)

And I guess `CLS_ATTR_COOKING` is no longer needed then

Reply:
Yes it was to much for mypy (metaclass + dynamic creation of `attrs`). In fact it  is the attrs plugins that does not support it.